pkg/core/blockchain.go

```diff
@@ -1294,6 +1295,14 @@ func (bc *Blockchain) verifyTxAttributes(tx *transaction.Transaction) error {
 			if uint64(tx.NetworkFee+tx.SystemFee) < req.GasForResponse {
 				return fmt.Errorf("%w: oracle tx has insufficient gas", ErrInvalidAttribute)
 			}
+		case transaction.NotValidBeforeT:
+			if !bc.config.P2PSigExtensions {
+				return errors.New("NotValidBefore attribute was found, but P2PSigExtensions are disabled")
```

Comment:
We may want to skip validation in this case? @roman-khimov

Reply:
We need to be compatible with the original protocol, so when no extensions are enabled it's an error to have any extensions.